src/main/java/com/example/HR_AppJava/HR_App/Job.java

```diff
@@ -9,13 +9,18 @@ public class Job {
     private String title;
     private int experiance;
 
-    public void setTitle(){
-        System.out.println("Enter title: ");
+    public Job(){
+        setTitle( title);
+        setExperience(experiance);
+    }
+
+    public void setTitle(String title){
+        System.out.print("Enter title: ");
         this.title = scanner.nextLine();
     }
 
-    public void setExperience(){
-        System.out.println("Enter experience: ");
+    public void setExperience(int experiance){
```

Comment:
You don't need to pass a parameter to the modifier since you are getting the user input inside the method.
Typos: print (should be println)
The experience variable is misspelled, but its consistent so its technically fine as is.